docs/data/theme/sea-ocean-and-coast/_data.yaml

```diff
@@ -1,5 +1,5 @@
 title: Sea, Ocean, and Coast
-description: Browse sea, ocean, and coast data products.
+description: Analyse the features and changes of coastal, tidal, and intertidal zones.
```

Comment:
```suggestion
description: Provide insights into Australia's coastal environments and their change through time
```

Reply:
@benji-glitsos-ga The current wording here isn't ideal - there's not really such thing as a tidal environment, and intertidal environments are a subset of coastal environment...

Reply:
@robbibt My current wording was this:

> Explore Australia's dynamic coastal, tidal, and intertidal zones.

But based on your feedback, I could change it to:

> Explore Australia's dynamic coastal environments including tidal and intertidal zones.

Or:

> Explore Australia's dynamic coastal environments including intertidal zones.

Reply:
How about this? "intertidal zone" should be singular, and this category technically includes marine/ocean data too in the high and low tide composites

> Explore Australia's marine and coastal environments including dynamic coastlines and the intertidal zone.

Although I'd be happy with something super simple too:

> Explore Australia's dynamic marine and coastal environments.

Reply:
Nice - I'll go with the first one.